index.js

```diff
@@ -20,44 +20,16 @@ const defaultConfig = JSON.parse(JSON.stringify(configjson[
 
 config = rc(pjson.name, defaultConfig);
 
-if (!config.blockchain[0].hubContractAddress && config.blockchain[0].networkId === defaultConfig.blockchain[0].networkId) {
-    config.blockchain[0].hubContractAddress = configjson[
-        process.env.NODE_ENV &&
-        ['development', 'testnet', 'mainnet'].indexOf(process.env.NODE_ENV) >= 0 ?
-            process.env.NODE_ENV : 'development'].blockchain[0].hubContractAddress
-}
+// if (!config.blockchain[0].hubContractAddress && config.blockchain[0].networkId === defaultConfig.blockchain[0].networkId) {
+//     config.blockchain[0].hubContractAddress = configjson[
+//         process.env.NODE_ENV &&
+//         ['development', 'testnet', 'mainnet'].indexOf(process.env.NODE_ENV) >= 0 ?
+//             process.env.NODE_ENV : 'development'].blockchain[0].hubContractAddress
+// }
 
 (async () => {
     try {
-        const node = new OTNode({
-            autoUpdate: config.autoUpdate,
-            rpcPort: config.rpcPort,
-            whitelist: config.ipWhitelist,
-            network: {
-                implementation: new Libp2p({
-                    bootstrapMultiAddress: config.network.bootstrap,
-                }),
-                ranking: {
-                    'kad-identity': new KadIdentityRanking(),
-                },
-            },
-            data: new GraphDB({
-                repositoryName: 'node0',
-                username: config.graphDatabase.username,
-                password: config.graphDatabase.password,
-            }),
-            validation: new MarkleValidation(),
-            blockchain: new Blockchain({
-                networkId: config.blockchain[0].networkId,
-                hubContractAddress: config.blockchain[0].hubContractAddress,
-                publicKey: config.blockchain[0].publicKey,
-                privateKey: config.blockchain[0].privateKey,
-                rpcEndpoints: config.blockchain[0].rpcEndpoints,
-            }),
-            telemetryHub: config.telemetryHub,
-            logLevel: config.logLevel,
-            replicationFactor: 3,
-        });
+        const node = new OTNode();
```

Comment:
When creating new otnode object you are able to pass configuration that you need, but for this startup we don't need it since the node will load it's configuration once you call otnode.start(). You can look at ot-node.js initializeConfiguration method